liquidity-module/index.md

```diff
@@ -303,15 +408,19 @@ Add `username`s mnemonic passphrase and hit enter. You will see something simila
 
 ## Create a Liquidity Pool
 
-To create a liquidity pool, enter the following command:
-Make sure to replace `longibchash` with the hash denom you received on the previous step. Replace `username` with your account username.
+Run the `liquidity create-pool` command to create a liquidity pool with two tokens: 1100000uphoton and 1500000ibc/denomhash. 
+
+Make sure to replace `ibc/denomhash` with the hash denom you received on the previous step and that you can verify in the [testnet supply](https://api.testnet.cosmos.network/cosmos/bank/v1beta1/supply). 
+
+Replace `username` with your account username.
 
 ```bash
-gaiad tx liquidity create-pool 1 1100000uphoton,1500000ibc/longibchash --from username --chain-id cosmoshub-testnet --gas-prices "0.025stake" --node https://rpc.testnet.cosmos.network:443 --gas 2000000 -y
+gaiad tx liquidity create-pool 1 1100000uphoton,1500000ibc/denomhash --from username --chain-id cosmoshub-testnet --gas-prices "0.025stake" --node https://rpc.testnet.cosmos.network:443 --gas 2000000 -y
```

Comment:
@kogisin I wasn't able to successfully create this pool. I verified the balances and token and IBC and relayer... I did very well up to this point, but then I got this 0 block height and no relayer action and no pool:

`raind tx liquidity create-pool 1 100000uphoton,100000ibc/C32E0052D50915581D261871CB35673B63637C0F1DD8C4A0EE9C2AC118F0737E --from cosmonaut61 --chain-id rain-cosmoshub-testnet --gas-prices 0.025stake --node https://rpc.testnet.cosmos.network:443 --gas 2000000`

this response (boooo no pool)
`{"height":"0","txhash":"6F6C00429FD3D31D005166D76695F98328376FF12678A781013042069A3C24F2","codespace":"sdk","code":4,"data":"","raw_log":"signature verification failed; please verify account number (443) and chain-id (cosmoshub-testnet): unauthorized","logs":[],"info":"","gas_wanted":"0","gas_used":"0","tx":null,"timestamp":""}`

Reply:
This error returns when inputting an incorrect account or chain id. I'll first follow along the tutorial and see if it is reproducible.

Reply:
@barriebyron I followed along the tutorial and I can't reproduce the same error. I instead encountered `insufficient balances for pool creation fee` error since the faucet only provides `10000000stake` while `PoolCreationFee` requires `100000000stake`. From the command line you input, I can only assume that you input the incorrect chain id. Unless you create a chain that has `rain-cosmoshub-testnet` chain id, it should be `cosmoshub-testnet`. But you would encounter the insufficient balances error unless you fauceted enough times to get enough balance (more than 10 times) from the `cosmoshub-testnet` chain.

References: 
- [Transaction that I encountered for the insufficient balances for pool creation fee error](https://api.testnet.cosmos.network/cosmos/tx/v1beta1/txs/00CFF298DFD918582095DC30D76BEA3AFCBA72304F767E73E527706B87E73361)
- [Look up pool creation fee](https://api.testnet.cosmos.network/tendermint/liquidity/v1beta1/params)

Reply:
After I fauceted my account enough to create a pool, I successfully created a liquidity pool. Here are the tx results.
- [Successfuly Tx](https://api.testnet.cosmos.network/cosmos/tx/v1beta1/txs/8711D0558727040B8A501AD512018449F1F00E8D2203B7CC6847D1664E3F0E10)
- [Pool 11](https://api.testnet.cosmos.network/tendermint/liquidity/v1beta1/pools)